maro/cli/inspector/cim_dashboard.py

```diff
@@ -0,0 +1,473 @@
+import math
+import os
+
+import numpy as np
+import pandas as pd
+
+import altair as alt
+import maro.cli.inspector.dashboard_helper as common_helper
```

Comment:
use ``isort --settings-path .github/linters/tox.ini`` to reformat the imports 

for imports under ``maro/cli/inspector/`` use relative imports like:
``from .prams import CIMItemOption, ...``